cpp/velox/shuffle/VeloxShuffleWriter.cc

```diff
@@ -1549,14 +1549,14 @@ arrow::Status VeloxShuffleWriter::splitFixedWidthValueBuffer(const velox::RowVec
       uint32_t numRows, const std::vector<std::shared_ptr<arrow::Buffer>>& buffers) {
     SCOPED_TIMER(cpuWallTimingList_[CpuWallTimingMakeRB]);
     if (codec_ == nullptr) {
-      return makeUncompressedRecordBatch(numRows, buffers, writeSchema(), payloadPool_.get());
+      return makeUncompressedRecordBatch(numRows, buffers, writeSchema(), options_.memory_pool);
```

Comment:
This is an unexpected change. We had shuffle writer memory pool separated into payloadPool_ and partitionBufferPool_, to help count the allocation of separately. Please refer to #3159 .

Reply:
Thanks, already fixed.